data/human/free worlds 0 prologue.txt

```diff
@@ -1218,11 +1277,43 @@ mission "FW Bounty 3"
 	on complete
 		payment 500000
 		"assisted free worlds" ++
-		log "Earned a good reputation with the Free Worlds militia by taking on missions to hunt down some dangerous pirates."
+		log "Defeated the Silverhawk, an Osprey armed with alien technology, and earned a good reputation with the Free Worlds militia by hunting down other dangerous pirates."
 		dialog `The militia officer who asked you to hunt down the <npc> contacts you soon after you land. "You've done us a great service," he says. "I'll be telling the Council how helpful you were." He hands you a payment of <payment>.`
 
 
 
+# Because these missions can also be failed by having "chosen sides", this mission handles the failure messages for FW Katya.
+mission "FW Katya Failed"
+	non-blocking
+	landing
+	invisible
+	to offer
+		not "chosen sides"
+		or
+			"FW Katya 1: failed"
+			"FW Katya 1B: failed"
+			"FW Katya Alt 2: failed"
+			"FW Katya 2: failed"
+			"FW Katya 2B: failed"
+			"FW Katya 2C: failed"
+			"FW Katya 3: failed"
+			"FW Katya 4: failed"
+			"FW Katya 5: failed"
+			"FW Katya 5B: failed"
+			"FW Katya 5C: failed"
+			"FW Katya 6: failed"
+			"FW Katya 6B: failed"
+			"FW Katya 6C: failed"
+			"FW Katya 7: failed"
+			"FW Katya 7B: failed"
+			"FW Katya 7C: failed"
+	on accept
```

Comment:
Any particular reason for using `on accept` over `on offer`?
```suggestion
	on offer
```

Reply:
So the `dialog phrase` doesn't show "accept" and "decline".